packages/babel/package.json

```diff
@@ -48,35 +52,35 @@
   "peerDependencies": {
     "@babel/core": "^7.0.0",
     "@types/babel__core": "^7.1.9",
-    "rollup": "^1.20.0||^2.0.0"
+    "rollup": "^1.20.0||^2.0.0||^3.0.0"
   },
   "peerDependenciesMeta": {
+    "rollup": {
+      "optional": true
+    },
```

Comment:
How Rollup becomes an optional peer dep here? 😅 it seems like a pretty hard requirement to have Rollup installed together with this plugin. Unless we are thinking about tools that bundle/wrap Rollup itself

Reply:
There were complaints from people using Vite + pnpm as then, Rollup is only a transitive dependency and apparently, pnpm does complain then. And technically, we do not depend on Rollup itself, just on its API contract. And there are tools out there that just kinda support the API contract these days.

Reply:
>  Unless we are thinking about tools that bundle/wrap Rollup itself
And you are right, there ARE tools that do that, thinking of Stencil here.